.github/workflows/cabal.yml

```diff
@@ -0,0 +1,22 @@
+on: [push]
+name: Cabal
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        ghc: ['8.10.7', '9.0.2', '9.2.7', '9.4.4']
+        cabal: ['3.8.1.0']
+        os: [ubuntu-latest]
```

Comment:
Nit: convert these to YAML-style lists (with `-`) to be consistent with the Stack file?